bin/main.rb

```diff
@@ -0,0 +1 @@
+p 'Hello World!'
```

Comment:
- [ ] Please follow the [indicated guide in Pathwright](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) To make your file executable
![image](https://user-images.githubusercontent.com/53256139/90189063-2685a080-dd82-11ea-91a3-efb88bff9568.png)